reconcile/glitchtip/reconciler.py

```diff
@@ -27,9 +28,16 @@ def _reconcile_projects(
     ) -> list[Project]:
         """Reconcile organization projects."""
         organization_projects = list(current_projects)
```

Comment:
`organization_projects` return value is not used, is there a reason to calculate and maintain it?

Reply:
See my comment below about the usage of `project_diff`. The return of `organization_projects` makes [testing](https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/test/glitchtip/test_glitchtip_reconciler.py#L155) easier.

Reply:
can be discussed in https://github.com/app-sre/qontract-reconcile/pull/3730#discussion_r1288230385 since they are related.